base/src/org/eevolution/process/GenerateCostDetail.java

```diff
@@ -393,10 +394,13 @@ else if (MPPCostCollector.COSTCOLLECTORTYPE_ActivityControl.equals(costCollector
             whereClause.append(" AND TRUNC(").append(MCostDetail.COLUMNNAME_DateAcct).append(")<=?");
             parameters.add(getDateAcctTo());
         }
+        String orderBy = " ORDER BY lowlevel desc, M_Product_ID ,  TRUNC( DateAcct ) , docstatuslevel," + 
```

Comment:
The order by clause is very complex and increase the response time for the database server, please review if this could be simplified

see this examples:

Query with order simplified:
![image](https://user-images.githubusercontent.com/1847863/113893056-a5599e80-9794-11eb-8aef-0efd8c697be4.png)
![image](https://user-images.githubusercontent.com/1847863/113893108-b60a1480-9794-11eb-8145-d457d5f305c6.png)

Query with order complex:
![image](https://user-images.githubusercontent.com/1847863/113893451-0aad8f80-9795-11eb-942b-c9605f9092e8.png)

![image](https://user-images.githubusercontent.com/1847863/113893411-008b9100-9795-11eb-8374-6b3a00da016d.png)

How do you see, the cost is increased from 1.23 to 1.26